v3/src/models/shared/shared-case-metadata.ts

```diff
@@ -46,6 +62,12 @@ export const SharedCaseMetadata = SharedModel
     // true if passed the id of a hidden attribute, false otherwise
     isHidden(attrId: string) {
       return self.hidden.get(attrId) ?? false
+    },
+    getLowColor(attrId: string) {
+      return self.numericColors.get(attrId)?.lowColor ?? kDefaultLowAttributeColor
+    },
+    getHighColor(attrId: string) {
+      return self.numericColors.get(attrId)?.highColor ?? kDefaultHighAttributeColor
```

Comment:
How about adding `getColorRange` which returns a `[lowColor, highColor]` array, since that seems to be what some clients want?